### PR TITLE
[MAD-14075] Upgrade EE to version 7.4.5

### PR DESCRIPTION
**Pull request by davemednick**
post-rebase

### CODE REVIEW

File: expressionengine/7/ExpressionEngine-7.4.5.tar.gz.md5 [hunk 1]
```diff
@@ -0,0 +1 @@
+MD5 (ExpressionEngine-7.4.5.tar.gz) = 4192d67cd3a809648d1b0e8e324f7aef
```
Comment (mpedrummer) [inline expressionengine/7/ExpressionEngine-7.4.5.tar.gz.md5]: This content differers from the 7.3.12 above, which is just the MD5. No idea if that's important, but it is different.

File: magento/2/latest-version [hunk 2]
```diff
@@ -1 +1 @@
-2.4.6-p4
\ No newline at end of file
+2.4.6-p4
```
Comment (mpedrummer) [inline magento/2/latest-version]: Should this change be here? Pretty sure we patched this to p5 earlier this week

**Review by mpedrummer — CHANGES_REQUESTED** [review]

File: magento/2/latest-version (hunk 2 above)
Comment (davemednick) [reply magento/2/latest-version]: is that a pending PR? master has 2.4.6-p4 in this file https://github.com/nexcess/app-sources/blob/master/magento/2/latest-version

Comment (mpedrummer) [reply magento/2/latest-version]: Wait, now I'm not sure - is this repo just irrelevant for Magento?

**Comment by nortoh** [comment]
We'll need to verify if this is breaking still with the error `PHP Fatal error:  Uncaught Error: Undefined constant "SELF"`. This was already present for the 7.3.8 version of EE

**Review by mpedrummer — APPROVED** [review]

**Review by nortoh — APPROVED** [review]

**Pull request merged**